kernel_gateway/gatewayapp.py

```diff
@@ -238,6 +245,10 @@ def init_webapp(self):
         # Redefine handlers off the base_url path
         handlers = []
         if self.api == 'notebook-http':
+            #Register the NotebookDownloadHandler if configuration allows
+            if self.allow_notebook_download:
+                handlers.append((r'/api/source', NotebookDownloadHandler, {'path': self.seed_uri}))
```

Comment:
Check with @Lull3rSkat3r about `/api`. We might have decided somewhere in the swagger work that we'd use the `/_api` for magic endpoints that we could expose without interfering with the API exposed by the notebook itself.